conf/layer.conf

```diff
@@ -9,4 +9,4 @@ BBFILE_COLLECTIONS += "readonly-rootfs-overlay"
 BBFILE_PATTERN_readonly-rootfs-overlay = "^${LAYERDIR}/"
 BBFILE_PRIORITY_readonly-rootfs-overlay = "6"
 
-LAYERSERIES_COMPAT_readonly-rootfs-overlay = "sumo"
+LAYERSERIES_COMPAT_readonly-rootfs-overlay = "zeus"
```

Comment:
Instead of replacing, this should be a list, so the layer stays compatible with older versions as well.